samples/XeroPrivate.gs

```diff
@@ -1,6 +1,6 @@
 /**
- * This sample that demonstrates how to configured the library for the Xero API,
- * when using a private application. Although the Xero documentation calls it a
+ * This sample demonstrates how to configure the library for the Xero API, when
```

Comment:
minor nit: I think we don't need a comma here

Reply:
Done!